src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -1518,6 +1534,97 @@ public void shouldReturnErrorResponseIfAllInstancesFailed()
     assertThat(instances.size(), is(0));
   }
 
+  /**
+   * Test case for instanceStatusUpdatedDateTrigger.sql trigger.
+   */
+  @Test
+  public void shouldSetStatusUpdatedDate() throws Exception {
+    UUID id = UUID.randomUUID();
+    JsonObject instanceToCreate = smallAngryPlanet(id)
+      .put("statusId", getOtherInstanceType().getId().toString());
+
+    createInstance(instanceToCreate);
+
+    Response createdInstance = getById(id);
+    assertThat(createdInstance.getJson().getString(STATUS_UPDATED_DATE_PROPERTY),
+      nullValue());
+
+    JsonObject replacement = instanceToCreate.copy()
+      .put("statusId", getCatalogedInstanceType().getId().toString());
+
+    JsonObject updatedInstance = updateInstance(replacement).getJson();
+
+    assertThat(updatedInstance
+        .getString(STATUS_UPDATED_DATE_PROPERTY), withinSecondsBeforeNow(seconds(2)));
+  }
+
+  /**
+   * Test case for instanceStatusUpdatedDateTrigger.sql trigger.
+   */
+  @Test
+  public void shouldOverrideStatusUpdatedDate() throws Exception {
```

Comment:
What does `override` mean in this context, is this test about the status updated date changing when the status changes?

Reply:
this test about updating instance status twice.

Reply:
@bohdan-suprun It is about the `status updated date` being updated when the status changes for a second time?

If so, maybe something like `shouldChangeStatusUpdatedDateOnSubsequentStatusChanges`?